composer.json

```
@@ -11,10 +11,14 @@
         }
     ],
     "autoload": {
-        "psr-4": { "ConnectionManager\\Extra\\": "src" }
+        "psr-4": {
+            "ConnectionManager\\Extra\\": "src"
```

If you're going to update this line, I suggest adding a trailing directory slash for consistency reasons:

```suggestion
            "ConnectionManager\\Extra\\": "src/"
```

I would also suggest moving the autoload/autoload-dev sections below the require/require-dev sections for consistency reasons. WDYT?

---

Yess, seems fine. I changed it, thanks @clue 👍